### PR TITLE
[MIG] [11.0] attachment_base_synchronize

### PR DESCRIPTION
**Pull request by thomaspaulb**

### CODE REVIEW

File: attachment_base_synchronize/data/cron.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        <field name='name'>Run Attachments Metadata</field>
+        <field name='interval_number'>30</field>
+        <field name='interval_type'>minutes</field>
+        <field name="numbercall">-1</field>
+        <field name="active">False</field>
+        <field name="doall" eval="False" />
+        <field name="model_id" ref="model_ir_attachment_metadata"/>
+        <field name="state">code</field>
+        <field name="code">model.run_attachment_metadata_scheduler()</field>
+        <field name="args">([])</field>
```
Comment (mlaitinen) [inline attachment_base_synchronize/data/cron.xml]: I went through this code and it LGTM. Just remove this `args` line and the Travis build will go green.

**Review by mlaitinen — CHANGES_REQUESTED** [review]

File: attachment_base_synchronize/data/cron.xml (hunk 1 above)
Comment (mlaitinen) [reply attachment_base_synchronize/data/cron.xml]: By Travis I meant Runbot of course ..

Comment (thomaspaulb) [reply attachment_base_synchronize/data/cron.xml]: Let's see if that will do it,

**Review by mlaitinen — APPROVED** [review]

**Review by florian-dacosta — APPROVED** [review]

**Comment by rousseldenis** [comment]
@OCA/core-maintainers

**Comment by NL66278** [comment]
@hbrunn Another candidate for merger. Approved since a longe time ago..., assing all checks...

**Pull request merged**